de.metas.fresh/de.metas.fresh.base/src/main/jasperreports/de/metas/docs/sales/order/report_details_v2.jrxml

```diff
@@ -457,7 +458,23 @@
 		<band height="12">
 			<property name="com.jaspersoft.studio.unit.height" value="pixel"/>
 			<textField isStretchWithOverflow="true" pattern="" isBlankWhenNull="true">
-				<reportElement key="textField-24" mode="Transparent" x="59" y="0" width="290" height="12" isPrintWhenDetailOverflows="true" forecolor="#000000" backcolor="#FFFFFF" uuid="74c7b578-d4a3-4e2e-8b6e-c25782bce4b0"/>
+				<reportElement key="textField-24" mode="Transparent" x="59" y="0" width="290" height="12" isRemoveLineWhenBlank="true" isPrintWhenDetailOverflows="true" forecolor="#000000" backcolor="#FFFFFF" uuid="74c7b578-d4a3-4e2e-8b6e-c25782bce4b0"/>
+				<box rightPadding="3">
+					<topPen lineWidth="0.0" lineStyle="Solid" lineColor="#000000"/>
+					<leftPen lineWidth="0.0" lineStyle="Solid" lineColor="#000000"/>
+					<bottomPen lineWidth="0.0" lineStyle="Solid" lineColor="#000000"/>
+					<rightPen lineWidth="0.0" lineStyle="Solid" lineColor="#000000"/>
+				</box>
+				<textElement>
+					<font fontName="SansSerif" size="9" isBold="false"/>
+				</textElement>
+				<textFieldExpression><![CDATA[($F{documentnote} != null && !$F{documentnote}.isEmpty()) ? $F{documentnote}  :  null]]></textFieldExpression>
```

Comment:
suggestion: generally test for `$F{documentnote} != null && !$F{documentnote}.trim().isEmpty()` because the user might have left some spaces or an enter in the field..
wdyt?